mappings/net/minecraft/item/Items.mapping

```diff
@@ -484,7 +502,7 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_8490 PURPUR_PILLAR Lnet/minecraft/class_1792;
 	FIELD field_8491 DANDELION Lnet/minecraft/class_1792;
 	FIELD field_8494 GOLD_BLOCK Lnet/minecraft/class_1792;
-	FIELD field_8495 SPRUCE_TRAPDOOR Lnet/minecraft/class_1792;
+	FIELD field_8495 INFESTED_COBBLESTONE Lnet/minecraft/class_1792;
```

Comment:
Does matcher not match the actual field to the new one?

Reply:
matcher seems to have fucked up on this front, i have fixed all the errors as far as i saw